apps/admin-gui/src/app/facilities/pages/resource-detail-page/resource-assigned-members/resource-assigned-members.component.html

```diff
@@ -1,16 +1,33 @@
 <h1 class="page-subtitle">{{'RESOURCE_DETAIL.ASSIGNED_MEMBERS.TITLE' | translate}}</h1>
 <perun-web-apps-refresh-button (refresh)="refreshTable()"> </perun-web-apps-refresh-button>
+
+<mat-form-field class="mr-2 fix-select" *ngIf="includeConsents">
+  <mat-label>{{'RESOURCE_DETAIL.ASSIGNED_MEMBERS.FILTER_CONSENT_STATUS' | translate}}</mat-label>
+  <mat-select (closed)="changeConsentStatuses()" [formControl]="consentStatuses" multiple>
+    <mat-select-trigger>
+      {{displaySelectedConsentStatuses()}}
+    </mat-select-trigger>
+    <mat-option
+      class="resource-theme"
+      *ngFor="let consentStatus of consentStatusList"
+      [value]="consentStatus"
+      >{{consentStatus | memberStatus}}</mat-option
+    >
```

Comment:
Prettier struggles to format this nicely.
```suggestion
      [value]="consentStatus">
      {{consentStatus | memberStatus}}
    </mat-option>
```